### PR TITLE
[zh-cn] sync reference/consistent-list-from-cache.md

### PR DESCRIPTION
**Pull request by RifeWang**
### Issue

Closes: #

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md [hunk 1]
```diff
@@ -9,8 +9,26 @@ stages:
...
+If older etcd version is provided Kubernetes will automatically detect it and fallback to serving consistent reads from etcd.
+Progress notifications ensure watch cache is consistent with etcd while reducing
+the need for resource-intensive quorum reads from etcd.
+
+See the Kubernetes documentation on [Semantics for **get** and **list**](/docs/reference/using-api/api-concepts/#semantics-for-get-and-list) for more details.
 -->
-允许 API 服务器从缓存中提供一致的 list 操作。
+通过直接从其 watch 缓存中响应一致的 **list** 请求，增强 Kubernetes API 服务器的性能，改善可扩展性和响应时间。
+为了从缓存中响应一致的 list，Kubernetes 需要使用较新的 etcd 版本（v3.4.31+ 或 v3.5.13+），
+这些版本包含了对 watch 进度请求功能的修复。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md:30]:
```suggestion
这些版本包含了对监视 watch 进度请求特性的修复。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md [hunk 2]
```diff
@@ -9,8 +9,26 @@ stages:
...
+如果使用的是较旧的 etcd 版本，Kubernetes 将自动检测并回退到从 etcd 中提供一致性读。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md:31]:
```suggestion
如果使用的是较旧的 etcd 版本，Kubernetes 将自动检测并回退到使用 etcd 完成一致性读操作。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md [hunk 3]
```diff
@@ -9,8 +9,26 @@ stages:
...
+进度通知确保 watch 缓存与 etcd 保持一致，同时减少了对 etcd 资源密集型仲裁读取的需求。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md:32]:
```suggestion
进度通知确保监视缓存与 etcd 保持一致，同时减少了对 etcd 资源密集的、带仲裁的读取的需求。
```

**Comment by RifeWang** [comment]
dup with https://github.com/kubernetes/website/pull/48062

**Pull request closed without merging**